### ISSUE TITLE
Idea: Fluid Transmutation

### ISSUE THREAD

**Issue by MaPePeR** [sinkillerj/ProjectE#428]
We already assign EMC to Fluids, so to take that to the next level there could be an 'alchemical tank' that could be used to transmute form EMC to a fluid of choice or be used to transmute fluids to items/other fluids.

(Idea from IRC)

**Comment by tankinnwow** [comment]
Yes, This is the BEST idea ever. I would live to see this happen transmuting items is cool but a lot of mods not use liquids for crafting and other things so.... why not

**Comment by Nova001Sol** [comment]
This. All of this.

**Comment by Phyn** [comment]
I have had the majority of the work for this done for quite a while and would be happy to finish and PR it.
